CountryApp/CountryApp/AccountManager/AccountManager.h

```diff
@@ -0,0 +1,18 @@
+//
+//  AccountManager.h
+//  CountryApp
+//
+//  Created by Artak Martirosyan on 5/21/18.
+//  Copyright © 2018 Artak. All rights reserved.
+//
+
+#import "Account.h"
+
+@interface AccountManager : NSObject
+
++ (id)sharedManager;
+
+- (BOOL) accountExistWithUsername:(NSString *) userName andPassword:(NSString *) pass;
```

Comment:
@amartiro if the method check whether an account exists or not, or in general if the method checks for some condition's true or false, it is a good practice to name that method with questioning style. It should be something like this: doesAccountExistWithUsername:andPass or isAccountExistsWithUserName:andPass